examples/http-server/main.go

```diff
@@ -41,6 +41,7 @@ func setupCoraza() error {
 	`); err != nil {
 		return err
 	}
+	fmt.Println("Coraza setup done.")
```

Comment:
Since this is sort of a real-world example, I would probably not include this line, don't think people will generally add a log line when configuring each component in an app